pkg/spoke/addon/configuration.go

```diff
@@ -0,0 +1,159 @@
+package addon
+
+import (
+	"crypto/sha256"
+	"crypto/x509/pkix"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	addonv1alpha1 "github.com/open-cluster-management/api/addon/v1alpha1"
+	certificatesv1beta1 "k8s.io/api/certificates/v1beta1"
+	"k8s.io/apimachinery/pkg/util/sets"
+)
+
+const (
+	installationNamespaceAnnotation = "installNamespace"
```

Comment:
these shoudl be namespaced in your API group.  `addon.something.something.io/installNamespace`

Reply:
Updated.